engine/inputdata/blueprint.ftl

```diff
@@ -21,6 +21,238 @@
     [/#if]
 [/#macro]
 
+[#function getBlueprintAttributes ]
```

Comment:
This might be better as getBlueprintConfiguration in line with the the collections of configuration objects you are merging in the function